boxsdk/object/item.py

```diff
@@ -5,11 +5,56 @@
 import json
 
 from .base_object import BaseObject
+from boxsdk.config import API
 
 
 class Item(BaseObject):
     """Box API endpoint for interacting with files and folders."""
 
+    @classmethod
```

Comment:
Preflight is a little different than other API calls in a sense that you can do preflight when trying to update an existing file on Box or uploading a new file. So it doesn't make sense to make a normal method tie to an object. But it can be a staticmethod, if that's why you're asking.